tests/filecheck/mlir-conversion/with-mlir/dialects/llvm/llvm_func.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: mlir-opt %s --mlir-print-op-generic | xdsl-opt --print-op-generic | filecheck %s
+
+"builtin.module"() ({
+  "llvm.func"() ({
+  }) {CConv = #llvm.cconv<ccc>, function_type = !llvm.func<void (i64, ...)>, linkage = #llvm.linkage<external>, sym_name = "printf", visibility_ = 0 : i64} : () -> ()
+
+  %0 = "test.op"() : () -> i64
+  llvm.call @printf(%0) : (i64) -> ()
```

Comment:
Personally, I think printf carries too much meaning to just use as a seemingly random name while testing. Why not call it `external_func`?

Reply:
This is pretty much the printf function signature though, I think it's neat that we can (almost) represent it now (and have it round-trip)

Reply:
For the record, we can represent the exact signature `i32 (llvm.ptr, ...)` but it doesn't round-trip because the llvm function type's custom printing changes it to `i32 (ptr, ...)` which our parser is unable to comprehend

Reply:
Okay, I am fine with leaving it  in👍